resources/todomvc/architecture-examples/vue/README.md

```diff
@@ -1,24 +1,37 @@
-# vue
+# Speedometer 3.0: TodoMVC: Vue
 
-## Project setup
-```
-npm install
-```
+## Description
 
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
+This application uses Vue.js 3.2.47 to implement a todo application.
+
+-   [Vue.js](https://vuejs.org/) provides efficient MVVM data bindings with a simple and flexible API. It uses plain JavaScript object models, DOM-based templating and extendable directives and filters..
+
+## Implementation details
+
+Vue.js is focused on the ViewModel layer of the MVVM pattern. It connects the View and the Model via two way data bindings. Actual DOM manipulations and output formatting are abstracted away into Directives and Filters.
+
+MVC:\
+Model: maintains the data and behavior of an application\
+View: displays the model in the ui\
+Container: serves as an interface between view & model components
+
+## Built steps
+
+To build the static files, this application utilizes webpack. It minifies and optimizes output files and copies all necessary files to a `dist` folder.
```

Comment:
What do you think of adding the command used to build?

Reply:
✅